projects/fsastorefrontlib/src/cms-components/banner/enriched-responsive-banner.component.html

```diff
@@ -1,11 +1,41 @@
 <ng-container *ngIf="component.data$ | async as data">
-  <div class="enriched-banner">
+  <div *ngIf="configStyles?.textBox" class="enriched-banner">
     <div class="image-wrapper">
       <cx-media [container]="data.media"></cx-media>
     </div>
-    <div class="enriched-banner-text">
-      <span class="enriched-banner-title">{{ data.headingText }}</span>
-      <span class="enriched-banner-details">
+    <div
+      class="enriched-banner-text"
+      [ngStyle]="bannerStyles"
+      [ngClass]="{
```

Comment:
If there are so many combinations, maybe move them to model (ts) and resolve them with switch case maybe, or something like that?

Reply:
On second thought could this be resolved in model as a property object and then just passed to [ngStyle]?

Reply:
I am not sure for `ngStyle`, we said it will be difficult for customer to override styles.

Reply:
Sorry, my bad, I meant `ngClass`